insights-notification-schemas-java/src/main/resources/schemas/Action.json

```diff
@@ -0,0 +1,123 @@
+{
```

Comment:
What happens to schema versioning? Will we still support that?

Reply:
Ok. From what I can see in `Parser`, it doesn't seem too hard to implement versioning later, so I'm fine with the absence of versioning for now.